onnxmltools/convert/coreml/shape_calculators/NonMaxSuppression.py

```diff
@@ -0,0 +1,17 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ...common._registration import register_shape_calculator
+from ...common.data_types import FloatTensorType, Int64TensorType
+from ...common.utils import check_input_and_output_numbers
+
+
+def calculate_non_max_suppression_output_shapes(operator):
```

Comment:
@vinitra It seems to me that this function is missing an implementation, and is the missing piece of this PR. Do you have any guidance for the community on where to get started finishing this up?

Reply:
Hi @chrisballinger! Thanks for your interest, and taking this on. I've addressed some of your review comments above. The remaining work:
- unit test
- shape inference calculations

I don't have the bandwidth for this currently, but would be happy to review PRs of anyone who wants to take this on.

Reply:
Great! Do you have links to documentation or examples for where someone might get started? I found this for CoreML: https://apple.github.io/coremltools/coremlspecification/sections/FeatureEngineering.html#nonmaximumsuppression

Reply:
Right, that's the link I was originally using from the CoreML side. There's also the original proto in coremltools, which has helpful comments:
https://github.com/apple/coremltools/blob/master/mlmodel/format/NonMaximumSuppression.proto
The relevant operator from Opset 11 on the ONNX side is here: 
https://github.com/onnx/onnx/blob/master/docs/Operators.md#NonMaxSuppression
For Opset 10, you can find the ONNX operator documentation below:
https://github.com/onnx/onnx/blob/master/docs/Changelog.md#NonMaxSuppression-10

Reply:
This is super helpful, thank you so much!